.github/actions/get-pull-number/action.yml

```
@@ -0,0 +1,42 @@
+---
+name: get-pull-number
+description: Get Pull Number
+inputs:
+  token:
+    type: string
```

```suggestion
  token:
    type: string
```
This needs to be passed as secrets
https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow